packages/engine/lib/error/src/frame.rs

```diff
@@ -16,6 +16,10 @@ use std::error::Error;
 use crate::provider::{self, Demand, Provider};
 use crate::Context;
 
+/// A [`Frame`] needs to implement all of the dependent traits so it's captured type needs these
+/// traits as well.
+///
+/// This is a convenient trait alias to avoid writing down each trait explicitly every time.
```

Comment:
Does this doc-comment appear on the other traits if under different cfgs, or do we have to copy paste it or something?

Reply:
Interesting, I didn't know that. I don't have the docs generated at the moment but does it show all definitions of the trait alias and the feature flags required for which one is active? Or does it just show the nightly one..

Reply:
Oh, sorry, this might have been misleading: We are only using the nightly compiler because of the toolchain file to generate the docs, also docs.rs is using the nightly compiler. This does not apply generally

Reply:
Right but the error crate won't only be generated on nightly? I suspect people will want to be able to see nightly docs and stable docs? If that's not the case or isn't really convention then I don't mind.

Just want to know what happens if I run `cargo doc` on the error lib itself outside of hEngine. (We're also likely to move it out of the folder it's in soon aren't we?)